discovery/store.go

```diff
@@ -221,26 +231,42 @@ func createPresentationRecord(serviceID string, timestamp Timestamp, presentatio
 	return &newPresentation, nil
 }
 
-// get returns all presentations, registered on the given service, starting after the given timestamp.
-// It also returns the latest timestamp of the returned presentations.
-// This timestamp can then be used next time to only retrieve presentations that were added after that timestamp.
-func (s *sqlStore) get(serviceID string, startAt Timestamp) ([]vc.VerifiablePresentation, *Timestamp, error) {
+// get returns all presentations, registered on the given service, starting after the given tag.
+// It also returns the latest tag of the returned presentations.
+// This tag can then be used next time to only retrieve presentations that were added after that tag.
+func (s *sqlStore) get(serviceID string, tag *Tag) ([]vc.VerifiablePresentation, *Tag, error) {
```

Comment:
just noticed that the `get` does not check if the service is still in the configured service list. So if control migrated from A to B, but clients still query A, they will always get an outdated list?

Reply:
failure on service layer: `Module.Get` checks this. SQL store doesn't do any checks the service layer can do.

I found out that check used the wrong map (all service definitions, instead of those of the server), so I'll fix that and add a test.